src/control/controlcompressingproxy.cpp

```diff
@@ -0,0 +1,39 @@
+#include "control/controlcompressingproxy.h"
+
+#include "moc_controlcompressingproxy.cpp"
+
+// Event queue compressing proxy
+CompressingProxy::CompressingProxy(QObject* parent)
+        : QObject(parent) {
+}
+
+// This function is called recursive by QCoreApplication::sendPostedEvents, until no more events are in the queue.
+// When the last event is found, QCoreApplication::sendPostedEvents returns for the processQueuedEvents() instance of this event,
+// and m_recursiveSearchForLastEventOngoing is set to false, while returning true itself.
+// All previous started instances of processQueuedEvents() will return false in consequence,
+// because the return value depends on the member variable and not on the stack of the instance.
+stateOfprocessQueuedEvent CompressingProxy::processQueuedEvents() {
+    m_recursiveSearchForLastEventOngoing = true;
```

Comment:
In extreme conditions, this replaces a stalled controller with a crash of Mixxx due to a stack overflow caused by a repeated CO updates fired from the real time audio thread. 
We assume, that we have only a few recursions here.
We should verify is by making this a counter and exit if the counter in unexpected high. Like 10 or a such. 
In such an extrem situations we will receive every 10th update, which sound reasonable.

Reply:
I implemented a saveguard for the stack. But it's not possible to interupt the processing of the event queue, once started. The only solution was to delete the remaining events from queue in extreme overload conditions.
I tested on my system that the stack is enough for about 500 recursions. I set the limit to a quarter of it. This should be a value with sufficient guard band to both sides. I expect, that it's never reached in practice.
This is covered by a new tescase and works on all platforms.